class-tgm-plugin-activation.php

```diff
@@ -2155,10 +2155,14 @@ function load_tgm_plugin_activation() {
 		}
 	}
 
-	if ( did_action( 'plugins_loaded' ) ) {
-		load_tgm_plugin_activation();
-	} else {
-		add_action( 'plugins_loaded', 'load_tgm_plugin_activation' );
+	if( function_exists( 'did_action' ) ) {
```

Comment:
Whitespace between `if` and `(` is needed.

Reply:
hey @GaryJones i'll make that change, actually ironically `ABSPATH` does get defined before composer dependancies